### PR TITLE
added note 

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5941

### CODE REVIEW

File: windows/deployment/windows-autopilot/windows-autopilot-requirements.md [hunk 1]
```diff
@@ -101,6 +101,9 @@ To provide needed Azure Active Directory (automatic MDM enrollment and company b
 - [Intune for Education subscriptions](https://docs.microsoft.com/intune-education/what-is-intune-for-education), which include all needed Azure AD and Intune features.
 - [Azure Active Directory Premium P1 or P2](https://azure.microsoft.com/services/active-directory/) and [Microsoft Intune subscriptions](https://www.microsoft.com/cloud-platform/microsoft-intune) (or an alternative MDM service).
 
+>[!NOTE]
```
Comment (illfated) [inline windows/deployment/windows-autopilot/windows-autopilot-requirements.md]: MarkDown indent compatibility spacing:
```suggestion
> [!NOTE]
```

File: windows/deployment/windows-autopilot/windows-autopilot-requirements.md [hunk 2]
```diff
@@ -101,6 +101,9 @@ To provide needed Azure Active Directory (automatic MDM enrollment and company b
...
+>In case of using Microsoft 365 subscriptions, you still need to [assign Intune licenses to the users](https://docs.microsoft.com/intune/fundamentals/licenses-assign)
```
Comment (illfated) [inline windows/deployment/windows-autopilot/windows-autopilot-requirements.md]: @JohanFreelancer9 : May I request your recommendation for this sentence structure?
```suggestion
> When using Microsoft 365 subscriptions, you still need to [assign Intune licenses to the users](https://docs.microsoft.com/intune/fundamentals/licenses-assign)
```

Comment (JohanFreelancer9) [reply windows/deployment/windows-autopilot/windows-autopilot-requirements.md]:
```suggestion
> Even when using Microsoft 365 subscriptions, you still need to [assign Intune licenses to the users](https://docs.microsoft.com/intune/fundamentals/licenses-assign).
```

**Comment by mypil** [comment]
@greg-lindsay - PR has been copy edited and is ready for final review. This is based on issue #5941.

Thank you.

cc: @kenwith

**Pull request merged**